powerspikegg/computation_models/match/BUILD

```diff
@@ -41,6 +41,14 @@ py_library(
     srcs = ["train.py"],
     visibility = ["//visibility:public"],
     deps = [
-        "@pydep_tensorflow//:library",
+        ":tensorflow",
     ],
 )
+
+alias(
```

Comment:
Ack. Please think about a clean way to include this. I might also take a look later.